db/migrate/20210206104409_remove_schools_for_foreign_users.rb

```
@@ -0,0 +1,13 @@
+class RemoveSchoolsForForeignUsers < ActiveRecord::Migration
+  def up
+    User.find_each do |user|
+      if user.country_code != "NZ" && user.school
+        user.school = nil
+        user.save
```

Shouldn't `user.save` be `user.save!` to fail loudly on errors?